tavern/ent/schema/job.go

```diff
@@ -29,19 +29,37 @@ func (Job) Fields() []ent.Field {
 // Edges of the Job.
 func (Job) Edges() []ent.Edge {
 	return []ent.Edge{
-		edge.To("createdBy", User.Type).
-			Required().
-			Unique(),
 		edge.To("tome", Tome.Type).
+			Comment("Tome that this job will be executing").
 			Required().
 			Unique(),
-		edge.To("tasks", Task.Type),
+		edge.To("bundle", File.Type).
+			Annotations(
+				entgql.Skip(entgql.SkipMutationCreateInput),
+			).
+			Comment("Bundle file that the executing tome depends on (if any)").
+			Unique(),
```

Comment:
Is this the file resources used inside the tome?

Reply:
Yessir o7